conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /                          app.Routes
+->         /iv-orchestration          app.Routes
```

Comment:
This should be "/" (as it was before) which would mean:
internal URLs will be `https://iv-orchestration.protected.mdtp/session/search`
external URLs will be `https://api.service.hmrc.gov.uk/individuals/iv-orchestration/session/search`

Reply:
But it is usual for internal services to have the service name as a context route. Then the mdtp routes files would route on this basis. The way it was done before I think reflected this?

Reply:
Technically this is fine but it will result in URLs:
internal URLs will be https://iv-orchestration.protected.mdtp/iv-orchestration/session/search
external URLs will be https://api.service.hmrc.gov.uk/individuals/iv-orchestration/iv-orchestration/session/search
The external one looking a bit odd as it duplicates "iv-orchestration"
I don't believe there should be any APIs that have endpoints like this